tests/python/relay/test_op_level6.py

```diff
@@ -88,7 +88,8 @@ def verify_argsort(shape, axis, is_ascend, dtype, is_dyn=False):
             verify_argsort((2, 3, 4), axis=0, is_ascend=False, dtype=dtype, is_dyn=is_dyn)
             verify_argsort((1, 4, 6), axis=1, is_ascend=True, dtype=dtype, is_dyn=is_dyn)
             verify_argsort((3, 5, 6), axis=-1, is_ascend=False, dtype=dtype, is_dyn=is_dyn)
-            verify_argsort((3, 2000, 6), axis=1, is_ascend=False, dtype=dtype, is_dyn=is_dyn)
+            verify_argsort((3, 6000, 6), axis=1, is_ascend=False, dtype=dtype, is_dyn=is_dyn)
+            verify_argsort((12000, 8, 8), axis=0, is_ascend=True, dtype=dtype, is_dyn=is_dyn)
             verify_argsort((1, 122640), axis=1, is_ascend=False, dtype=dtype, is_dyn=is_dyn)
```

Comment:
Can we add a workload from your benchmark script, that was causing an issue before?